### PR TITLE
Add tags validation in create-news

### PR DESCRIPTION
**Pull request by Gusvarov**

### CODE REVIEW

File: src/app/service/eco-news/create-eco-news.service.ts [hunk 1]
```diff
@@ -72,21 +73,21 @@ export class CreateEcoNewsService {
...
+ 
+    formData.append('image', this.files[0].file, this.files[0].file.name);
+    formData.append('addEcoNewsDtoRequest', JSON.stringify(body));
+
+    console.log(JSON.stringify(body), this.files[0].file, this.files[0].file.name);
 
     this.httpOptions.headers.set('Authorization', `Bearer ${this.accessToken}`);
-    return this.http.post<NewsResponseDTO>(`${this.url}/econews`, body, this.httpOptions);
+    // this.httpOptions.headers.append('Content-Type', 'multipart/form-data;boundary='+Math.random());
+    // this.httpOptions.headers.append('Accept', 'application/json');
```
Comment (RomanDeveloperAcc) [inline src/app/service/eco-news/create-eco-news.service.ts:90]: Please, remove the commented code if it is not necessary

File: src/app/component/eco-news/create-news/create-news.component.css [hunk 2]
```diff
@@ -205,6 +209,10 @@ button {
     border: none;
 }
 
+.red {
```
Comment (KimAT25) [inline src/app/component/eco-news/create-news/create-news.component.css:212]: please change the name of class to more understandable

File: src/app/component/eco-news/create-news/create-news.component.html [hunk 3]
```diff
@@ -40,6 +40,9 @@ <h3>Pick tags for news</h3>
                                 [ngClass]="{
                                     'filters-color': filter.isActive
                                 }">{{ filter.name }}</button>
+                                <p [ngClass]="{
+                                    'red': isFilterValidation
+                                }">Only 3 tags can be added</p>
```
Comment (KimAT25) [inline src/app/component/eco-news/create-news/create-news.component.html:45]: please put tag`s ends (</button> and </p> in new line for better readability.
And put content inside tags also in new line

File: src/app/service/eco-news/create-eco-news.service.ts [hunk 4]
```diff
@@ -72,21 +73,21 @@ export class CreateEcoNewsService {
   }
 
   public sendFormData(form, language): Observable<NewsResponseDTO> {
-    const body: NewsDTO = {
-      "imagePath": form.value.source,
+    const body: any = {
```
Comment (KimAT25) [inline src/app/service/eco-news/create-eco-news.service.ts:76]: please change type from any

File: src/app/service/eco-news/create-eco-news.service.ts [hunk 5]
```diff
@@ -48,6 +48,7 @@ export class CreateEcoNewsService {
   public setTranslationByLang(language: string, translations: NewsModel): void {
     this.translations[language].text = translations.text;
     this.translations[language].title = translations.title;
+    console.log(this.files[0].file);
```
Comment (KimAT25) [inline src/app/service/eco-news/create-eco-news.service.ts:51]: please delete console.log

File: src/app/service/eco-news/create-eco-news.service.ts [hunk 6]
```diff
@@ -72,21 +73,21 @@ export class CreateEcoNewsService {
...
-      ]
+      "text": form.value.content,
+      "title": form.value.title,
     };
+    let formData:FormData = new FormData();
+ 
+    formData.append('image', this.files[0].file, this.files[0].file.name);
+    formData.append('addEcoNewsDtoRequest', JSON.stringify(body));
+
+    console.log(JSON.stringify(body), this.files[0].file, this.files[0].file.name);
```
Comment (KimAT25) [inline src/app/service/eco-news/create-eco-news.service.ts:86]: please delete console.log

File: src/app/service/eco-news/create-eco-news.service.ts (hunk 1 above)
Comment (Gusvarov) [reply src/app/service/eco-news/create-eco-news.service.ts]: fixed

File: src/app/component/eco-news/create-news/create-news.component.css (hunk 2 above)
Comment (Gusvarov) [reply src/app/component/eco-news/create-news/create-news.component.css]: fixed

File: src/app/component/eco-news/create-news/create-news.component.html (hunk 3 above)
Comment (Gusvarov) [reply src/app/component/eco-news/create-news/create-news.component.html]: fixed

File: src/app/service/eco-news/create-eco-news.service.ts (hunk 4 above)
Comment (Gusvarov) [reply src/app/service/eco-news/create-eco-news.service.ts]: fixed

File: src/app/service/eco-news/create-eco-news.service.ts (hunk 5 above)
Comment (Gusvarov) [reply src/app/service/eco-news/create-eco-news.service.ts]: fixed

File: src/app/service/eco-news/create-eco-news.service.ts (hunk 6 above)
Comment (Gusvarov) [reply src/app/service/eco-news/create-eco-news.service.ts]: fixed

**Pull request closed without merging**